requirements.txt

```diff
@@ -151,3 +151,6 @@ django-notifications-hq==1.2 \
     --hash=sha256:4a7afdc506a15aad89434fb9fd41b611ba2a75bc0f5ba15179b5998c5181a703
 django-model-utils==2.0.3 \
     --hash=sha256:fae8d517590d99730ed3cfc3fdf9378e0b3815325a8911b963241ee8e9761fd7
+compare-locales==1.2.3 \
```

Comment:
For now, using the recent commit hash is good. We'll cut a 2.0 release of compare-locales soon, though, and then we should use that published release.